src/main/scala/mesosphere/marathon/api/v2/GroupNormalization.scala

```diff
@@ -0,0 +1,29 @@
+package mesosphere.marathon
+package api.v2
+
+import mesosphere.marathon.state.PathId
+
+object GroupNormalization {
+
+  def partialUpdateNormalization(conf: MarathonConf): Normalization[raml.GroupPartialUpdate] = Normalization { update =>
+    update.copy(enforceRole = Some(effectiveEnforceRole(conf, update.enforceRole)))
+  }
+
+  def updateNormalization(conf: MarathonConf, id: PathId): Normalization[raml.GroupUpdate] = Normalization { update =>
+    // Only update if this is not a scale or rollback
+    if (update.version.isEmpty && update.scaleBy.isEmpty) {
+      if (id.parent.isRoot) {
+        update.copy(enforceRole = Some(effectiveEnforceRole(conf, update.enforceRole)))
```

Comment:
So, this should probably be the original group role if one present so that it won't be default when the original is not the default. What do you think @meichstedt?

Reply:
Or was this for apps only? IMHO `PUT` should always be an override.